src/Tests/Peachpie.Test/Program.cs

```diff
@@ -36,4 +36,4 @@ static void Main(string[] args)
             }
         }
     }
-}
```

Comment:
can we revert this change please :)

Reply:
Resolved https://github.com/peachpiecompiler/peachpie/pull/593/commits/a5f7ebace380b9103c52ca3a06707f3eb80f1122